mmdet/models/dense_heads/rpn_head.py

```diff
@@ -23,14 +24,32 @@ class RPNHead(AnchorHead):
     def __init__(self,
                  in_channels,
                  init_cfg=dict(type='Normal', layer='Conv2d', std=0.01),
+                 num_convs=1,
                  **kwargs):
+        self.num_convs = num_convs
         super(RPNHead, self).__init__(
             1, in_channels, init_cfg=init_cfg, **kwargs)
 
     def _init_layers(self):
         """Initialize layers of the head."""
-        self.rpn_conv = nn.Conv2d(
-            self.in_channels, self.feat_channels, 3, padding=1)
+        if self.num_convs > 1:
+            rpn_convs = []
+            for i in range(self.num_convs):
+                if i == 0:
+                    in_channels = self.in_channels
+                else:
+                    in_channels = self.feat_channels
+                rpn_convs.append(
+                    ConvModule(
+                        in_channels,
+                        self.feat_channels,
+                        3,
+                        padding=1,
+                        inplace=False))
```

Comment:
Why inplace should be set to False？

Reply:
This way of writing will result in the existence of two Relu ops. Should it be improved?

Reply:
> This way of writing will result in the existence of two Relu ops. Should it be improved?

should be fine.